confidence.py

```diff
@@ -237,7 +237,62 @@ def loads(*strings):
     return Configuration(*(yaml.load(string) for string in strings))
 
 
-def read_envvars(name):
+def read_xdg_config_dirs(name, extension):
+    """
+    Read from files found in XDG-specified system-wide configuration paths,
+    defaulting to ``/etc/xdg``. Depends on ``XDG_CONFIG_DIRS`` environment
+    variable.
+
+    :param name: application or configuration set name
+    :param extension: file extension to look for
+    :return: a `.Configuration` instance with values read from XDG-specified
+        directories
+    """
+    # find optional value of ${XDG_CONFIG_DIRS}
+    config_dirs = environ.get('XDG_CONFIG_DIRS')
+    if config_dirs:
+        # PATH-like env vars operate in decreasing precedence, reverse this path set to mimic the end result
+        config_dirs = reversed(config_dirs.split(path.pathsep))
+    else:
+        # XDG spec: "If $XDG_CONFIG_DIRS is either not set or empty, a value equal to /etc/xdg should be used."
+        config_dirs = ['/etc/xdg']
```

Comment:
on standard windows machines this will never be found

Reply:
this will not be a problem, it will be catched in the next test

Reply:
All the read-methods check for existence before doing anything exciting, non-existent directories shouldn't cause any issues (even when the paths make no sense on some operating systems :)).